src/reducers/versions.spec.tsx

```diff
@@ -117,12 +120,48 @@ describe(__filename, () => {
 
       expect(state).toEqual({
         ...initialState,
+        expandedPaths: getParentFolders(version.file.selected_file),
         versionInfo: {
           [version.id]: createInternalVersion(version),
         },
       });
     });
 
+    it('prunes expandedPaths when loading version info', () => {
+      const version = { ...fakeVersion, id: nextUniqueId() };
+      const newVersion = { ...fakeVersion, id: nextUniqueId() };
+      const _getPrunedExpandedPaths = jest.fn(getPrunedExpandedPaths);
+      const _reducer = createReducer({ _getPrunedExpandedPaths });
+
+      const state = _reducer(undefined, actions.loadVersionInfo({ version }));
+      _reducer(state, actions.loadVersionInfo({ version: newVersion }));
+
+      expect(_getPrunedExpandedPaths).toHaveBeenCalledWith(
+        state.expandedPaths,
```

Comment:
> I am not very sure to have `state.expandedPaths` ...

👍 

This is a good value to use for the assertion because the `expandedPaths` contents don't matter here. The assertion is just checking that the value gets passed to `getPrunedExpandedPaths()`.